releases/v20.1.0-alpha.20200210.md

```diff
@@ -0,0 +1,351 @@
+- Three new CLI commands `cockroach auth-session login`, `cockroach auth-session list` and `cockroach auth-session logout` are now provided to facilitate the management of web sessions. The command `auth-session login` also produces a HTTP cookie which can be used by non-interactive HTTP-based database management tools. It also can generate such a cookie for the `root` user, who would not otherwise be able to do so using a web browser. [#43872][#43872]
+
+### Admin UI changes
+
+- Decommissioned node history page is now viewable on a dedicated page. This reduces the amount of information on the [Cluster Overview](../v20.1/admin-ui-cluster-overview-page.html) page [#42817][#42817]
+- Execution Latency graph is now renamed to "KV Execution Latency" [#43290][#43290]
+- Redesigned the Cluster Overview page [#43552][#43552]
+- We previously introduced a fix on the Admin UI to prevent non-admin users from executing queries. However, this inadvertently caused certain pages requiring table details not to display. This issue has now been resolved. [#44167][#44167]
+- Non-admin users can now use the [Jobs](../v20.1/admin-ui-jobs-page.html) page of the Admin UI to see their own jobs. [#44345][#44345]
+
+### Bug fixes
+
+- Previously, when vectorized execution engine was used with `vectorize=experimental_on`, CockroachDB could incorrectly report some values as NULL. This has now been fixed. [#43785][#43785]
+- When casting a string to bytes, octal escapes greater than `\377` will now generate an error, rather than silently wrapping around. [#43806][#43806]
+- Job can be running but shown in state pending. [#43814][#43814]
+- On Linux machines, we now respect the available memory limit as dictated by the cgroup limits which apply to the cockroach process. [#43137][#43137]
+- Previously, CockroachDB would return incorrect results for some aggregate functions when used as window functions with non-default window frame. This is now fixed. Note that MIN, MAX, SUM, AVG, and "pure" window functions (i.e., non-aggregates) were not affected. [#39308][#39308]
+- Previously, CockroachDB could return an internal error when running a query with a CAST operation (`:::`) in some cases when using the vectorized execution engine. This is now fixed. [#43857][#43857]
+- Previously, a query shutdown mechanism could fail to fully cleanup the infrastructure when the query was executed via the vectorized engine and the query plan contained wrapped row-by-row processors (in v19.2, this applies to Lookup joins and Index joins). This is now fixed. [#43579][#43579]
```

Comment:
This and other mentions of "vectorized" should probably link to https://www.cockroachlabs.com/docs/dev/vectorized-execution.html